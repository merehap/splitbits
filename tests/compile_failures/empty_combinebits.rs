use splitbits::*;

fn main() {
    combinebits!();
}
