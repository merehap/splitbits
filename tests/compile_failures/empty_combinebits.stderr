error: proc macro panicked
 --> tests/compile_failures/empty_combinebits.rs:4:5
  |
4 |     combinebits!();
  |     ^^^^^^^^^^^^^^
  |
  = help: message: combinebits! must take at least one argument (the template).
